docs/containers.md

```diff
@@ -1,6 +1,6 @@
 Running in Containers
 =====================
-`AMPLFI` also provides a container builds available via the github container repository
+`AMPLFI` also provides container builds available via the github container repository
```

Comment:
Theres actually only one container now (the `data` and `train` containers should be removed) So let's make this 
`provides a container build`